### ISSUE TITLE
Need Show Description Properties Or Method Correct

### ISSUE THREAD

**Issue by chuongmep** [weianweigan/RevitLookupWpf#21]
**Is your feature request related to a problem? Please describe.**
this is idea extend form #20 because we have file json contains description inside export from project [RevitAPIDocGen](https://github.com/chuongmep/RevitAPIDocGen)

**Describe the solution you'd like**
Replace description it with description exist
Example : use **Autodesk.Revit.DB.DirectContext3D.IndexPrimitive** to check and return **Description** is `This is a description`
```json
{
        "Title":"IndexPrimitive",
        "Keywords":"Autodesk.Revit.DB.DirectContext3D.IndexPrimitive class",
        "APIName":"AllMembers.T:Autodesk.Revit.DB.DirectContext3D.IndexPrimitive",
        "Description":"This is a description",
        "Namespace":"Autodesk.Revit.DB.DirectContext3D",
        "Guid":"005eb672-881e-8295-6e96-8efc82177b05",
        "Type":"class"
    },
```

**Describe alternatives you've considered**

**Additional context**
![image](https://user-images.githubusercontent.com/31106432/154276686-27500a11-8fba-413c-9a6e-7dda1c829428.png)

**Comment by weianweigan** [comment]
Do you means show Description in control's tooltip?

**Comment by chuongmep** [comment]
> Do you means show Description in control's tooltip?
@weianweigan my idea is create new windows and use **context menu** same with Search online help to show description, some thing like that ,this can help us to minimize the waiting time and only use it when needed, you have any idea ? 
 
![image](https://user-images.githubusercontent.com/31106432/154429857-c32d34ec-ebe1-43e3-a9c7-2b9eb704ffdb.png)

**Comment by weianweigan** [comment]
I agree with you.This can help us to minimize the waiting time. If we show description in tooltip,it will cost some time to query json file.

**Comment by weianweigan** [comment]
I just add a apiname property to find revitinfo and made a window to show help.

![image](https://user-images.githubusercontent.com/25452811/154637247-0a6b198d-7229-4bde-b7d3-c41a01229f72.png)
